### PR TITLE
Update sips extension

### PR DESCRIPTION
**Pull request by arronhunt**
## Description

This change updates Stephen Kaplan's (@SKaplanOfficial) [Image Modification extension](https://www.raycast.com/HelloImSteven/sips) to show real previews of image filters. Currently only a list with descriptions is displayed, and adding a visual preview can help users make better decisions and see a preview of the output before committing to a change. 

## Screencast

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/sips/package.json [hunk 1]
```diff
@@ -48,15 +31,7 @@
...
       "description": "Rotate the selected images by the specified amount of degrees",
-      "mode": "no-view",
-      "arguments": [
-        {
-          "name": "degrees",
-          "placeholder": "Degrees",
-          "type": "text",
-          "required": true
-        }
-      ]
```
Comment (SKaplanOfficial) [inline extensions/sips/package.json:59]: I think the arguments were removed by mistake? They're needed for the command to be useful.

File: extensions/sips/package.json [hunk 2]
```diff
@@ -68,55 +43,19 @@
...
-          "type": "text",
-          "required": false
-        },
-        {
-          "name": "height",
-          "placeholder": "Height",
-          "type": "text",
-          "required": false
-        }
-      ]
```
Comment (SKaplanOfficial) [inline extensions/sips/package.json:85]: Same as above

File: extensions/sips/package.json [hunk 3]
```diff
@@ -68,55 +43,19 @@
...
       "description": "Scale the selected images by the provided factor",
-      "mode": "no-view",
-      "arguments": [
-        {
-          "name": "scaleFactor",
-          "placeholder": "Scale Factor",
-          "type": "text",
-          "required": true
-        }
-      ]
```
Comment (SKaplanOfficial) [inline extensions/sips/package.json:99]: Same as above

File: extensions/sips/package.json [hunk 4]
```diff
@@ -68,55 +43,19 @@
...
-          "type": "text",
-          "required": true
-        },
-        {
-          "name": "hexcolor",
-          "placeholder": "Color",
-          "type": "text",
-          "required": false
-        }
-      ]
```
Comment (SKaplanOfficial) [inline extensions/sips/package.json:119]: Same as above

**Review by SKaplanOfficial — CHANGES_REQUESTED** [review]
Really good change! Just some minor problems with the package.json, otherwise looks good!

**Comment by arronhunt** [comment]
Thanks @SKaplanOfficial, I didn't touch package.json myself so something must have happened during the fork 😟. Should be an easy fix though.

**Pull request merged**